tests/unit/lib/SuiteCRM/Robo/Commands/CodeCoverageCommandsTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+use \SuiteCRM\Robo\Plugin\Commands\CodeCoverageCommands;
+
+class CodeCoverageCommandsTest extends \Codeception\Test\Unit
```

Comment:
use state check please

Reply:
I have switched the unit to the \SuiteCRM\StateCheckerUnitAbstract